src/CentralPackageVersions/README.md

```diff
@@ -59,6 +59,8 @@ In some cases, you may need to override the version for a particular project.  T
 </Project>
 ```
 
+A `PackageReference` does not have to have a corresponding central version if it has a `VersionOverride`. If you want to require a central version for each `PackageReference` in your code base, set `DisableVersionOverridesWithoutCentralVersions` to `true`.
```

Comment:
The other control mechanism is named `EnablePackageVersionOverride` which defaults to `true`. 

I think we should name this new property `EnablePackageVersionOverrideWithoutCentralVersion` which also defauls to `true` and you set it to `false` to generate the error.  Seems like keeping it consistent will give a better user experience.